testing/src/main/java/org/motechproject/nms/testing/web/TestingController.java

```diff
@@ -29,4 +33,10 @@ public String sendEvent(@RequestParam(required = true) String subject) {
         return String.format("Sent MOTECH event: %s with empty params: %s", subject, eventParams.toString());
     }
 
+    @RequestMapping(value = "/sendSMS200", method = RequestMethod.POST, headers = { "Content-type=application/json" })
+    @ResponseStatus(HttpStatus.ACCEPTED)
```

Comment:
Yes, I understand that but you cannot call an API sendSMS200 and send status code 202 for it. That's bad practice. Please change.